db/database.go

```diff
@@ -191,10 +191,12 @@ func NewDatabaseContext(dbName string, bucket base.Bucket, autoImport bool, opti
 	}, options.CacheOptions, options.IndexOptions)
 	context.SetOnChangeCallback(context.changeCache.DocChanged)
 
-	if err = context.tapListener.Start(bucket, options.TrackDocs, func(bucket string, err error) {
-		context.TakeDbOffline("Lost TAP Feed")
-	}); err != nil {
-		return nil, err
+	if !(options.IndexOptions != nil && options.IndexOptions.Writer == false) {
```

Comment:
I don't believe we need to start the listener when options.IndexOptions.Writer=true - sg_accel should be managing its own feed now.